TbsCore/Tasks/LowLevel/SellOnAuctions.cs

```diff
@@ -87,7 +90,7 @@ public override async Task<TaskRes> Execute(Account acc)
                 var element = acc.Wb.Driver.FindElement(By.XPath(nodeItemXPath.XPath));
                 if (element == null) continue;
                 element.Click();
-                await Task.Delay(600);
+                await Task.Delay(rand.Next(1500, 2500));

```

Comment:
Maybe add acc delay here instead?

Reply:
no it should be over 1 sec, it should have its own settings cannot use together with acc.delay